rm-datasource/src/main/java/io/seata/rm/datasource/undo/AbstractUndoExecutor.java

```diff
@@ -117,9 +117,8 @@ public void executeOn(Connection conn) throws SQLException {
         if (IS_UNDO_DATA_VALIDATION_ENABLE && !dataValidationAndGoOn(conn)) {
             return;
         }
-        try {
-            String undoSQL = buildUndoSQL();
-            PreparedStatement undoPST = conn.prepareStatement(undoSQL);
+        String undoSQL = buildUndoSQL();
```

Comment:
ensure all the exception can be caught and throw sqlException

Reply:
+1